mkt/reviewers/views.py

```diff
@@ -277,7 +277,7 @@ def _review(request, addon, version):
     all_forms = [form, attachment_formset, testedon_formset]
 
     if version:
-        features_list = [unicode(f) for f in version.features.to_list()]
+        features_list = [unicode(f) for f in version.features.to_names()]
```

Comment:
I just realized I went through this though process and... did it already. So this can probably go, I'll give it a try.